src/metaUtils.cxx

```diff
@@ -1293,6 +1288,17 @@ bool MET_Read(METAIO_STREAM::istream &fp,
   return MET_IsComplete(fields);
   }
 
+std::string convert_ulonglong_to_string(MET_ULONG_LONG_TYPE val)
+  {
+  std::string result;
+  while (val > 0)
+    {
+    result = static_cast<char>((val % 10)+ '0') + result;
+    val /= 10;
+    }
+  return result;
+  }
+
```

Comment:
I have verified that this code works, but I am not fully sure that it is actually required, or very useful, as there is this previous conversion to double in the value field... the real test would be really large numbers, close to unsigned long long max, or close to double's digit limit...